core/field_image.ts

```diff
@@ -27,7 +27,6 @@ export class FieldImage extends Field<string> {
    * of the field.
    */
   private static readonly Y_PADDING = 1;
-  protected override size_: Size;
```

Comment:
What was this even doing??

Reply:
Nothing AFAICT; I suspect it was introduced by the automated tooling during the TS migration.